### ISSUE TITLE
Tie gets pixelated, when more than two notes are tied together

### ISSUE THREAD

**Issue by cyluxx** [opensheetmusicdisplay/opensheetmusicdisplay#884]
Hi,

When having more than two notes tied together, the first tie is rendered smooth, whereas all subsequent ties get pixelated:

![temp](https://user-images.githubusercontent.com/21027404/92819717-6499d280-f3c9-11ea-9d9e-81a3be8b75d8.png)

I observed this behaviour also at demos on the OSMD website, e.g. "Lee Actor - Prelude".

Is there the possibility, that two ties are overlapping and thus creating this effect? 

Appendix: This is the corresponding XML for the image above:
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE score-partwise PUBLIC "-//Recordare//DTD MusicXML 3.1 Partwise//EN" "http://www.musicxml.org/dtds/partwise.dtd">
<score-partwise version="3.1">
    <part-list>
        <score-part id="P1">
            <part-name>
                Music
            </part-name>
        </score-part>
    </part-list>
    <part id="P1">
        <measure number="1">
            <attributes>
                <divisions>
                    8
                </divisions>
                <key>
                    <fifths>
                        0
                    </fifths>
                </key>
                <time>
                    <beats>
                        4
                    </beats>
                    <beat-type>
                        4
                    </beat-type>
                </time>
                <clef>
                    <sign>
                        G
                    </sign>
                    <line>
                        2
                    </line>
                </clef>
            </attributes>
            <note>
                <pitch>
                    <step>
                        A
                    </step>
                    <octave>
                        4
                    </octave>
                </pitch>
                <duration>
                    32
                </duration>
                <type>
                    whole
                </type>
                <tie type="start"/>
                <notations>
                    <tied type="start"/>
                </notations>
            </note>
        </measure>
        <measure number="2">
            <note>
                <pitch>
                    <step>
                        A
                    </step>
                    <octave>
                        4
                    </octave>
                </pitch>
                <duration>
                    32
                </duration>
                <type>
                    whole
                </type>
                <tie type="stop"/>
                <tie type="start"/>
                <notations>
                    <tied type="stop"/>
                    <tied type="start"/>
                </notations>
            </note>
        </measure>
        <measure number="3">
            <note>
                <pitch>
                    <step>
                        A
                    </step>
                    <octave>
                        4
                    </octave>
                </pitch>
                <duration>
                    32
                </duration>
                <type>
                    whole
                </type>
                <tie type="stop"/>
                <notations>
                    <tied type="stop"/>
                </notations>
            </note>
            <barline location="right">
                <bar-style>
                    light-heavy
                </bar-style>
            </barline>
        </measure>
    </part>
</score-partwise>
```

**Comment by sschmidTU** [comment]
Yes, it looks like we are rendering two identical ties on top of each other.
The least we can do is not render the second tie if an identical one was already rendered, as a quick improvement.
It looks like our tie parsing has some general issues (#863, #712, #713), so the second step would be improving on tie and slur creation in general.

**Comment by sschmidTU** [comment]
Fixed in OSMD 1.4.3, ties are no longer created multiple times / pixelated.
![image](https://user-images.githubusercontent.com/33069673/150037563-6bb9724a-99f0-4f2d-9e4d-54399a3124cf.png)
[tie_pixelated_fixed.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/7893100/tie_pixelated_fixed.zip)
